### DISCUSSION TITLE
support api update / delete without primary key (using a filter)

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#5926] [Feature Requests]
For example to update a field inside a translation collection only knowing the main collection pk, filtering by _languages_code_

Hasura does it like this:

- https://hasura.io/docs/latest/graphql/core/api-reference/graphql-api/mutation.html#update-syntax
- https://hasura.io/docs/latest/graphql/core/api-reference/graphql-api/mutation.html#delete-syntax

**Comment by benhaynes** [comment]
Duplicate of: https://github.com/directus/directus/discussions/3119
